java/opendcs/src/main/java/decodes/db/DatabaseIO.java

```diff
@@ -399,6 +398,17 @@ public abstract void deletePresentationGroup( PresentationGroup pg )
 	public abstract Date getPresentationGroupLMT(PresentationGroup pg)
 		throws DatabaseException;
 
+	/**
+	 * If the presentation group referenced by groupId is used by one or more routing
+	 * specs, return a list of routing specs populated with only IDs and names. If groupId is not used,
+	 * return null.
```

Comment:
we shouldn't return null collections (and also the implementations don't return null)